src/plugins/index.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
```

Comment:
why you use require instead import? ESM?

Reply:
It cause errors even if I will define this file as .mjs with the es6 module i/e